src/Core/Mapping/RequestParameterMapping.php

```diff
@@ -217,7 +217,7 @@ protected function normalize($value, EndpointParameter $parameter, ITypeMapper $
 		}
 	}
 
-	protected function getMapper(string $type): ?ITypeMapper
+	public function getMapper(string $type): ?ITypeMapper
```

Comment:
If it's only for tests, use reflection.

Take a look at Liberator class (https://github.com/contributte/redis/blob/master/tests/cases/DI/RedisExtension.phpt#L136-L137).

Reply:
It is for tests but it make sense for me to have this public.